docs_src/examples/Ch-ExamplesSendingAndConsumingBinary.md

```diff
@@ -185,3 +185,49 @@ data inside the EdgeX Reading's `BinaryValue` field.
 Using this approach, an Event can be sent containing data containing an
 arbitrary, flexible structure. Use cases could be a Reading containing
 multiple images, a variable length list of integer read-outs, etc.
+
+### Creating A CBOR Payload For Use With PUT Commands
+
+To create a CBOR payload that can, for example, be used with `PUT` commands,
+we first need to create a CBOR decoder (from a given set of CBOR bytes),
+along with a CBOR code handle. We` decode into an EdgeX event where, as
```

Comment:
Can we update this to have a closing backtick?

Also, can we update this to reflect the proper data structure that is being encoded?